src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -664,7 +664,7 @@ public void run() {
                     for (BibEntry entry : entries) {
                         bes = entry;
                         // Store the old value:
-                        oldvals.put(bes, bes.getField(BibEntry.KEY_FIELD));
+                        oldvals.put(bes, bes.getCiteKey());
```

Comment:
Yeah, I got some failing tests due to empty cite keys and while investigating replaced these calls. Its some additional code cleanup and I did not saw any value in opening an extra PR for this commit.